$NetBSD: patch-af,v 1.1 2010/10/02 10:08:31 wiz Exp $

commit 7dbec1171386f182be77942ae43dc3684a96cfc4
Author: Mark Kretschmann <kretschmann@kde.org>
Date:   Sat Oct 2 09:06:11 2010 +0200

    Better workaround for wrong applet size on startup.

    This version also works in the case that Amarok was hidden to
    tray before starting, and it does not produce a jerking motion
    when starting in fullscreen mode.
    Thanks to Felix Geyer <debfx-pkg@fobos.de> for the idea.

    It might worth to include this patch in Amarok packages.

--- src/MainWindow.h.orig	2010-09-15 21:24:11.000000000 +0000
+++ src/MainWindow.h
@@ -148,6 +148,7 @@ class AMAROK_EXPORT MainWindow : public 
         void slotAddStream();
         void slotJumpTo();
         void showScriptSelector();
+        void resizeWindowHack();
 #ifdef DEBUG_BUILD_TYPE
         void showNetworkRequestViewer();
 #endif // DEBUG_BUILD_TYPE
